docs/howto/writing-migrations.txt

```diff
@@ -86,11 +87,39 @@ the respective field according to your needs.
 * Add the field on your model with ``default=...`` and ``unique=True``
```

Comment:
The idea is to create a generate recipe that can be applied for any field type, not just uuid. Is that unclear?